src/ServiceControl/HeartbeatMonitoring/HeartbeatsMonitor.cs

```diff
@@ -56,7 +58,7 @@ void Refresh(object _)
 
                 try
                 {
-                    timer.Change((int)TimeSpan.FromSeconds(5).TotalMilliseconds, -1);
+                    timer.Change(TimeSpan.FromSeconds(5), Timeout.InfiniteTimeSpan);
```

Comment:
Should we use [`PeriodicExecutor`](https://github.com/Particular/ServiceControl/blob/master/src/ServiceControl/Infrastructure/PeriodicExecutor.cs) in these startup tasks?